docker/mjml/Pipfile

```
@@ -0,0 +1,12 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+mjml-python = "*"
+
+[dev-packages]
+
+[requires]
+python_version = "3.13"
```

```suggestion
python_version = "3.12"
```
The Docker image inherits from the Python 3.12 tag, so please use the corresponding Python version in pip.